lib/xcactivitylog.rb

```diff
@@ -12,5 +12,7 @@ def self.parse_file(path:)
     unzipped_contents = Zlib.gunzip(contents)
 
     Parser.new(class_deserializer: {}).parse!(unzipped_contents)
+  rescue Zlib::GzipFile::Error => e
```

Comment:
I think this should be done by callers of this method -- there's no reason a library should be swallowing errors by default